nuxeo-jsf/nuxeo-platform-webapp-base/src/main/java/org/nuxeo/ecm/webapp/tree/BigFolderLeafFilter.java

```diff
@@ -31,13 +32,11 @@
 
     private static final long serialVersionUID = 1L;
 
-    public static final String BIG_FOLDER_FACET = "BigFolder";
-
```

Comment:
It *may* be used by customers, we don't know... So having one full release with it deprecated is better.